test/comm.jl

```diff
@@ -0,0 +1,17 @@
+if VERSION >= v"0.5-"
+    using Base.Test
+else
+    using BaseTestNext
+    const Test = BaseTestNext
+end
+
+import IJulia: init, Comm, comm_target
```

Comment:
No need to import `IJulia.init` if you are going to call it as a fully qualified name anyway.